notebooks/newsletter-agent.ipynb

```diff
@@ -1,866 +1,850 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    newsletter_tool = Tool.from_function(newsletter_pipeline_func)
</pre><p>we should use <code>create_tool_from_function</code></p><p><a href="https://docs.haystack.deepset.ai/docs/tool#create_tool_from_function" target="_blank">https://docs.haystack.deepset.ai/docs/tool#create_tool_from_function</a> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>